flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/Member.java

```diff
@@ -48,6 +50,7 @@ public static class MemberBuilder {
         private Object type;
         private String name;
         private String defaultValue;
+        private boolean optional = false;
```

Comment:
can simplify as `false` is the default value for boolean fields in Java
```suggestion
        private boolean optional;
```

Reply:
Thanks. Will do this another iteration